packages/components/src/alignment-matrix-control/cell.tsx

```diff
@@ -25,7 +26,10 @@ export default function Cell( {
 
 	return (
 		<Tooltip text={ tooltipText }>
-			<CompositeItem as={ CellView } role="gridcell" { ...props }>
+			<CompositeItem
+				id={ id }
+				render={ <CellView { ...props } role="gridcell" /> }
```

Comment:
I wonder if there's any difference in spreading `props` on `CellView` instead of `CompositeItem` ?

Reply:
I guess `props` may have some specific `CellView` custom props that could conflict with the props `CompositeItem` accept. Those should be passed directly to `CellView`. Usually, TypeScript will error when you do something wrong, which might have been the case here.